src/test/regress/sql/fkeys_between_local_ref.sql

```diff
@@ -278,6 +278,26 @@ BEGIN;
   ORDER BY tablename;
 ROLLBACK;
 
+BEGIN;
+  -- disable foreign keys to reference tables
+  SET LOCAL citus.enable_local_reference_table_foreign_keys TO false;
+  CREATE TABLE local_table_6 (col_1 INT PRIMARY KEY);
+
+  -- first convert local tables to citus local tables in graph
+  ALTER TABLE local_table_2 ADD CONSTRAINT fkey_11 FOREIGN KEY (col_1) REFERENCES reference_table_1(col_1) ON DELETE CASCADE;
+
+  CREATE TABLE local_table_5 (
+    col_1 INT UNIQUE REFERENCES local_table_6(col_1),
+    col_2 INT REFERENCES local_table_3(col_1),
+    FOREIGN KEY (col_1) REFERENCES local_table_5(col_1));
```

Comment:
as `enable_local_reference_table_foreign_keys` is set to false, none of those tables are citus local tables, so let's add one more line:

```sql
FOREIGN KEY (col_1) REFERENCES reference_table_1(col_1)
```

Reply:
ops, of course we should do that. Thanks for catching